/**:
  ros__parameters:
    voxel_size_x: 0.6
    voxel_size_y: 0.6
    voxel_size_z: 0.6
    max_queue_size: 5
